templates/chirp.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
```

Comment:
We need to be disciplined about the scope of the card. When working on a card about logins, it will be important not to introduce unrelated functionality, such as chirping.